python/cudf/cudf/core/dataframe.py

```diff
@@ -3516,10 +3496,8 @@ def quantiles(self, q=0.5, interpolation="nearest"):
         result = self._quantiles(q, interpolation.upper())
 
         if q_is_number:
-            result = result.transpose()
-            return Series(
-                data=result._columns[0], index=result.index, name=q[0]
-            )
+            result = result.to_pandas().transpose()
```

Comment:
This smells fishy...

Reply:
Quantiles results in a single row when `q` is a singular value. This row shouldn't be too long.

Reply:
```
if q_is_number:
    return Series(result.iloc[0], name=q[0])
```

Reply:
@shwina sorry, github hadn't refreshed. Can't use iloc either. :man_shrugging: